model/model.md

```diff
@@ -14,10 +22,18 @@ Note that all relationships are multiplicities unless specified.
 
 - A [Dataspace Authority](./terminology.md#dataspace-authority) manages one or more [Dataspaces](./terminology.md#dataspace). This will include [Participant](./terminology.md#participant) registration and may entail mandating business and/or technical requirements. For example, a [Dataspace Authority](./terminology.md#dataspace-authority) may require [Participants](./terminology.md#participant) to obtain some form of business certification. A [Dataspace Authority](./terminology.md#dataspace-authority) may also impose technical requirements such as support for the technical enforcement of specific usage policies.
 - A [Participant](./terminology.md#participant) is a member of one or more [Dataspaces](./terminology.md#dataspace). A [Participant](./terminology.md#participant) registers [Participant Agents](./terminology.md#participant-agent) that perform tasks on its behalf.
+- A [`Dataspace Registry` or `Dataspace Registration Service`](./terminology.md#dataspace-registration-service) manages the registered `Participant Agents` in a `Dataspace`. Registration and unregistration processes may be different depending on various aspects.
```

Comment:
I think this had been already added with #229. Please check again and prevent duplicates.